src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -53,6 +53,8 @@ public partial class MarkStep : IStep
 		protected List<TypeDefinition> _typesWithInterfaces;
 		protected List<MethodBody> _unreachableBodies;
 
+		readonly List<(TypeDefinition Type, MethodBody Body, Instruction Instr)> pending_isinst_instr;
```

Comment:
Nit: member fields start with `_` in this class - so to keep to consistent it should be `_pending_isinst_instr`.